lib/src/sql/statement.rs

```diff
@@ -247,4 +252,22 @@ mod tests {
 		let out = res.unwrap().1;
 		assert_eq!("CREATE test;\nCREATE temp;", format!("{}", out))
 	}
+
+	#[test]
+	fn show_table_changes() {
+		let sql = "SHOW CHANGES FOR TABLE test SINCE 123456";
+		let res = statement(sql);
+		assert!(res.is_ok());
+		let out = res.unwrap().1;
+		assert_eq!("SHOW CHANGES FOR TABLE test SINCE 123456", format!("{}", out))
+	}
+
+	#[test]
+	fn show_database_changes() {
+		let sql = "SHOW CHANGES FOR DATABASE SINCE 123456";
```

Comment:
Change Feed in this pull request does support reading the feed for the all the tables in a database via this query.

However, the write side does not support the database-wide change feed. That is, there's no facility to let you enable the database-wide change feed recording.

Maybe we need `DEFINE DATABSE CHANGEFEED 1h` or something like that implemented in an upcoming PR to enable the write side of the database change feed!